<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubPGSaleNorBuy</class>
 <widget class="QWidget" name="SubPGSaleNorBuy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Normal</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="PBRm">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>180</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBAdd">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>180</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBConfirm">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>248</y>
     <width>80</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSearch">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>108</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="bookListTable">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>9</y>
     <width>311</width>
     <height>131</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Title</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Amount</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>120</y>
     <width>45</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineAmount">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>140</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>31</width>
     <height>23</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="PBSearchBook">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="inventoryTable">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>159</y>
     <width>311</width>
     <height>401</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Title</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Amount</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="PBCancel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>250</y>
     <width>80</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBUpdate">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineName">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>53</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>53</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
